### PR TITLE
57110 Update phone number on alert

### LINKED ISSUE #57110: Update phone number on 526-EZ alert #49356
On hold - pending discussion on slack thread about the support hours around eVSS Helpdesk.
https://dsva.slack.com/archives/CNCEXNXK4/p1684342364382719

## Issue Description
As a Veteran I want to make sure that if there is some information that I need to get to the VA regarding my disability claim submission that I can get that information to the correct person.

# Background
On the 526-EZ form we make a check before the Veteran begins the form to make sure we can set up an EDIPI or a BIRLS ID for the Veteran because without one of these they cannot submit a claim. If we are unable to establish one of these we show the Veteran a message that we need more information shown here -

The correct phone number we need on this error message is 800-698-2411, that is the number for the Tier 1 Contact Center which can route issues like this to the EVSS help desk. They are available 24 hours a day, 7 days a week.

## Tasks
- [x] Update the phone number on the alert
- [ ] Notify Tier 1 Group of the change with added training
- [ ] Submit a PR for review and merge

## Acceptance criteria
- [x] The correct phone number now shows on the alert message referenced above
- [ ] Designer comments on this issue to approve page styling 
- [x] Unit tests pass
- [x] E2E tests cover current code and regression
- [x] End-to-end tests show 0 violations.
- [x] All axe checks pass
- [x] All screenreader checks pass

### PR DESCRIPTION
**Pull request by christinec-fftc**
## Summary
Update phone number and availability

On the 526-EZ form we make a check before the Veteran begins the form to make sure we can set up an EDIPI or a BIRLS ID for the Veteran because without one of these they cannot submit a claim. If we are unable to establish one of these we show the Veteran a message that we need more information.

The phone number number and content need to be updated. The correct phone number we need on this error message is 800-698-2411, that is the number for the Tier 1 Contact Center which can route issues like this to the EVSS help desk. They are available 24 hours a day, 7 days a week.

(Which team do you work for, does your team own the maintenance of this component?)
DBEx (Disability Benefits Experience) Team

## Related issue(s)
[#57110](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/57110)

## Testing done
_Describe what the old behavior was prior to the change_
Previous phone number is incorrect.

_Describe the steps required to verify your changes are working as expected_
1. go to form 526 `/disability/file-disability-claim-form-21-526ez/introduction` and file a new claim
2. log in as user+3 
3. start a new claim
4. alert will display

_Describe the tests completed and the results_
Manual test as noted above for desktop, mobile and VoiceOver were successful

Existing unit and e2e tests successful

## Screenshots
image
image

## What areas of the site does it impact?
Just this one alert on form 526

### Quality Assurance & Testing

- [n/a] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [n/a] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [n/a] Events are being sent to the appropriate logging solution
- [n/a] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

**Review by mchae-nava — APPROVED** [review]
LGTM! I see a linting warning for a deprecated import, but wondering if that concerns us at all since that wasn't touched

Noting also confirmed the manual test and voice over on mac

**Comment by christinec-fftc** [comment]
@mchae-nava @navaeddie I tried changing the import, but unfortunately it causes a bunch of test failures at the app level (Form526EZApp.unit.spec.jsx) which are out of scope for this story. Will leave it be for now until we can ramp up on that area of 526.

**Review by sethdarragile6 — APPROVED** [review]

**Review by navaeddie — APPROVED** [review]

**Review by adamwhitlock1 — APPROVED** [review]
Some feedback on tty link, but otherwise lgtm

File: src/applications/disability-benefits/all-claims/containers/MissingServices.jsx [hunk 1]
```diff
@@ -63,9 +63,9 @@ export const MissingId = ({ title }) => {
         We don’t have all of your ID information for your account. We need this
         information before you can {titleLowerCase(title)}. To update your
         account, please call Veterans Benefits Assistance at{' '}
-        <va-telephone contact={CONTACTS.VA_BENEFITS} /> (TTY:{' '}
+        <va-telephone contact={CONTACTS.HELP_DESK} /> (TTY:{' '}
         <va-telephone contact={CONTACTS['711']} />
```
[the remark this answers is not in the document]
Comment (adamwhitlock1) [reply src/applications/disability-benefits/all-claims/containers/MissingServices.jsx]: 👍🏼

**Comment by christinec-fftc** [comment]
Finally got updated content today. Ready for review once more.

**Review by mchae-nava — APPROVED** [review]
Clicked through, looks great! The pronunciation utility is pretty neat

**Comment by christinec-fftc** [comment]
Thanks for looking @navaeddie! Yes, I updated the screenshots with the latest changes as well. The phone number is the same, but the content changed a little.

**Pull request merged**